fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -363,13 +370,23 @@ fun BottomSheet(
                             .draggable(
                                 orientation = Orientation.Vertical,
                                 state = rememberDraggableState { delta ->
-                                    sheetState.performDrag(delta)
+                                    if(!enableSwipeDismiss && sheetState.offset.value != null && sheetState.offset.value!! >= (fullHeight - dpToPx(peekHeight) ) ){
```

Comment:
Add the check in the semantic modifier used for dismiss for talkback.

Reply:
Done, review the new commit